src/dxvk/dxvk_graphics.cpp

```diff
@@ -1108,7 +1128,7 @@ namespace dxvk {
 
       // Do not compile if this pipeline can be fast linked. This essentially
       // disables the state cache for pipelines that do not benefit from it.
-      if (this->canCreateBasePipeline(state))
+      if (!gplAsyncCache && !m_async && this->canCreateBasePipeline(state))
```

Comment:
Use solution from DXVK-GPLALL 2.7-3, which I took from Britt Yazel DXVK GPLAsync 2.7 patch.
```suggestion
      if (!gplAsyncCache && !m_async.load(std::memory_order_acquire) && this->canCreateBasePipeline(state))
```